components/sup-protocol/protocols/ctl.proto

```diff
@@ -104,6 +104,34 @@ message SvcLoad {
   optional sup.types.UpdateCondition update_condition = 17;
 }
 
+message SvcUpdate {
```

Comment:
It might not be too hard to do in a backwards-compatible way... we'd probably need to have the consumer of the messages attempt to read the new version, and then fall back to the older version.

I think the more complicated bit might be dealing with the fact that some fields are different, and have different defaults (specifically looking at the `group` field). It's not impossible, but does raise the complexity / impact of such changes a bit.